plugins/inputs/internet_speed/README.md

```diff
@@ -63,24 +64,25 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 It collects the following fields:
 
-| Name           | field name | type    | Unit |
-|----------------|------------| ------- | ---- |
-| Download Speed | download   | float64 | Mbps |
-| Upload Speed   | upload     | float64 | Mbps |
-| Latency        | latency    | float64 | ms   |
-| Jitter         | jitter     | float64 | ms   |
-| Location       | location   | string  | -    |
+| Name           | field name  | type    | Unit |
+|----------------|-------------|---------|------|
+| Download Speed | download    | float64 | Mbps |
+| Upload Speed   | upload      | float64 | Mbps |
+| Latency        | latency     | float64 | ms   |
+| Jitter         | jitter      | float64 | ms   |
+| Packet Loss    | packet_loss | float64 | pr   |
```

Comment:
pr is percent?

Reply:
In general, it has no unit because it is probability.

Reply:
The closer to 1 means the greater the chance you had packet loss and if it is exactly 1 then you had packet loss?

Reply:
> The closer to 1 means the greater the chance you had packet loss and if it is exactly 1 then you had packet loss?

1 means all packets loss.

Reply:
Maybe we can record the details of packet loss into the fields.

Contains three additional fields：
1. total number of packets received by the remote.
2. dup packet number.
3. the maximum index value received by the remote.

Reply:
> 1 means all packets loss.

That sounds like a percentage then. I'm fine with reporting a single field as a percentage, I just wanted to clarify the meaning of the values. The end user probably won't care as much about how many packets were sent.

Reply:
percentage = probability * 100%, so it's a probability :)

Reply:
changed

Reply:
Hmm I really don't like the term probability when you are reporting a hard-value that was observed once.

You are calculating that value by taking the total number of packets received and dividing it by total sent right? Why not multiple it by 100 and make it a percentage? If you look at ping output you get a % of packet loss and it seems that is what you are trying to show here.

Reply:
okay, i changed it.

Reply:
Thanks!